projects/core/src/i18n/i18n.module.ts

```diff
@@ -19,6 +41,12 @@ export class I18nModule {
         provideDefaultConfig(defaultI18nConfig),
         { provide: TranslationService, useExisting: I18nextTranslationService },
         ...i18nextProviders,
+        {
+          provide: APP_INITIALIZER,
```

Comment:
This should be CONFIG_INITIALIZER (instead of APP_INITIALIZER) that declares resolving `i18n.fallbackLang`

Reply:
I used `CONFIG_INITIALIZER` at the beginning, then it doesn't work. I checked all functions having `configInit.getStableConfig()...`, then I found all of them use `APP_INITIALIZER`.

Reply:
Interesting it didn't work. I'll check it.
Regardless, our intention is to initialize the `i18n.fallbackLang` config based on dynamic `context.langauge` config. APP_INITIALIZER cannot do it. Only CONFIG_INITIALIZER.
Yes, one CONFIG_INITIALIZER (i18n.fallbackLang) waits for another CONFIG_INITIALIZER (context) to resolve.

Reply:
It doesn't work on my local, maybe because I have `context.i18n` typo.